packages/vite-plugin-cloudflare/src/assets/router-worker.ts

```diff
@@ -0,0 +1,4 @@
+// @ts-ignore
+import routerWorker from '@cloudflare/workers-shared/dist/router-worker.mjs';
+
+export default routerWorker;
```

Comment:
This file is just to make the imports elsewhere in the code look cleaner?

Reply:
It's so that all the internal workers used by the plugin are built and imported in a consistent way. So this file is the entry for the router worker build:

https://github.com/flarelabs-net/vite-plugin-cloudflare/blob/ca5b0697b2914941e94483880b3f7c8775fc09e1/packages/vite-plugin-cloudflare/tsup.config.ts#L22